src/hotspot/share/opto/c2_globals.hpp

```diff
@@ -785,7 +785,9 @@
           range(0, max_juint)                                               \
                                                                             \
   product(bool, UseProfiledLoopPredicate, true,                             \
-          "Move predicates out of loops based on profiling data")           \
+          "Move checks with an uncommon trap out of loops based on "        \
+          "profiling data. "                                                \
+          "Requires UseLoopPredicate to be turned on (default).")           \
```

Comment:
Can you also update the comment for `UseLoopPredicate`? It seems outdated / wrong.

Now is:
`Generate a predicate to select fast/slow loop versions`

@chhagedorn do you have a good suggestion for what to put now?

Reply:
Good catch! It could be similar but without mentioning profiling data:
```
Move checks with an uncommon trap out of loops.
```